nnvm/python/nnvm/frontend/keras.py

```diff
@@ -479,9 +516,12 @@ def keras_op_to_nnvm(insym, keras_layer, outname, symtab):
     """
     if type(keras_layer).__name__ not in _convert_map:
         raise NotImplementedError("{} is not supported".format((type(keras_layer).__name__)))
-    ret = _convert_map[type(keras_layer).__name__](insym, keras_layer, symtab)
-    symtab.set_var(outname, ret)
+    sym = _convert_map[type(keras_layer).__name__](insym, keras_layer, symtab)
+    sym = _as_list(sym)
 
+    for t_idx, _sym in enumerate(sym):
```

Comment:
_sym is used as a local name of nnvm.symbol.  I'd suggest using a different name.